src/nwb_benchmarks/benchmarks/network_tracking_remote_file_reading.py

```diff
@@ -33,257 +34,231 @@
 )
 
 
```

Comment:
Wow, this diff is terrible. Basically I moved the skip to the method, renamed some benchmark methods to be more clear, and added docstrings